src/components/ScenarioParameters/ScenarioParameters.js

```diff
@@ -156,14 +164,19 @@ const ScenarioParameters = ({
 
   // You can use the context object to pass all additional information to custom tab factory
   const context = {
-    editMode,
     isDarkTheme,
+    isReadOnly: false,
   };
 
   useEffect(() => {
     discardLocalChanges();
     // eslint-disable-next-line
-  }, [currentScenarioData?.id]);
+  }, [scenarioId]);
+
+  useEffect(() => {
+    if (scenarioStatus === STATUSES.SUCCESS) reset({ ...getValues() });
+    // eslint-disable-next-line
+  }, [scenarioStatus]);
```

Comment:
`reset({ ...getValues() });` is already called when the scenario is launched, do we really need it here too?

Reply:
This reset is used to correctly handle parameters reset after save.
In fact, it is the reset during scenario launch which is now redundant.

Reply:
If we remove the whole block, then `scenarioStatus` and `currentScenario` can be removed from the hook as well, I think?